source/install/install-ubuntu-2004.rst

```diff
@@ -0,0 +1,23 @@
+.. _install-ubuntu-2004:
+
+=========================================
+Installing Mattermost on Ubuntu 20.04 LTS
+=========================================
+
+Install a production-ready Mattermost system on 1 to 3 machines.
```

Comment:
```suggestion
Install a production-ready Mattermost system on up to three machines.
```

Reply:
Out of scope of this PR: I'm not sure whether "machines" is a good term to use. I am 3/5 for using "servers". @jasonblais, thoughts?

Reply:
I'm 0/5 as well, but is the terminology used in other install guides.

Reply:
I think best to leave as-is for now then. I'll note it down as something to consider adjusting in future. Thanks!